modules/validations/almost_junction.js

```diff
@@ -148,6 +148,12 @@ export function validationAlmostJunction(context) {
         }));
       }
 
+      // changing the date range is always an option
+      fixes.push(new validationIssueFix({
+        icon: 'iD-operation-change-date-range',
+        title: t.append('issues.fix.change_date_range.title')
+      }));
```

Comment:
This additional fix is causing two tests to fail because there are more suggested fixes than expected. I updated the test in #142.